resources/wikia/modules/tracker.js

```diff
@@ -135,15 +135,19 @@
 		 */
 		function sendInternalTrackingEvent( event, data, timeout ) {
 			var head = document.head || document.getElementsByTagName( 'head' )[ 0 ] || document.documentElement,
-					script = document.createElement( 'script' ),
-					requestUrl = 'https://beacon.wikia-services.com/__track/special/' + encodeURIComponent( event ),
-					requestParameters = [],
-					p,
-					params,
-					userId = -1;
+				script = document.createElement( 'script' ),
+				requestUrl = 'https://beacon.wikia-services.com/__track/special/' + encodeURIComponent( event ),
+				requestParameters = [],
+				p,
+				params,
+				userId = -1,
+				wikiVariables = M && M.getFromHeadDataStore && M.getFromHeadDataStore('wikiVariables'),
+				trackingDimensions = M && M.getFromHeadDataStore && M.getFromHeadDataStore('trackingDimensions');
```

Comment:
`... || [];`
so you can use `window.wgArticleId || trackingDimensions[21]` in L170

Reply:
💎 
Changed.